### DISCUSSION TITLE
Funding - OpenXR integration

### DISCUSSION THREAD

**Discussion by JeffGillin** [bevyengine/bevy#8913] [General]
Hey all,
   I wanted to see if someone could put together a GoFund me, or similar, for specific cause, which is OpenXR integration.
   I've currently moved to Unity to learn more about the various technologies in XR field.
   Of course, I'm a Bevy guy and want to stay here, as I believe (strongly) that it's the best choice for game engine.
   I would personally be willing to donate $3K usd **for a switch** in direction to focus on OpenXR integration.
   Perhaps a GoFundMe for $60k, and if the mark is hit, a heavy focus is put on XR for several months, perhaps even having something ready for when the standalone headsets are released. I can think of SO many AR apps already.  IMHO, the XR field will be the VAST MAJORITY of 3d work in the future. After the tease of the next gen headsets coming out, I think lots would be willing to contribute to this cause.
Jeff

**Comment by pkupper** [comment]
Are you aware of the [Bevy Sponsorhip Pledge](https://bevyengine.org/sponsorship-pledge/)?  There are, for vey good reasons, some limitations in place regarding sponsorships and how they can affect project direction. That said, according to this pledge you are free to sponsor individuals to work on certain features but Maintainers and Project Leads will always prioritize according to community needs and not differentiate between sponsored and non-sponsored contributions.

**Reply by JeffGillin** [reply]
Yeah, but this is likely a relatively big effort, not sponsored by a single individual.
If we had something like a crowd fund for Bevy, I think we can really speed up the process of getting Bevy fully XR ready.
Perhaps a kickstarter or something could really bring in some major funds for bevy.
They could receive some training videos for their contribution.

**Reply by mockersf** [reply]
You can find a list of some of the contributors accepting sponsorship here: https://bevyengine.org/community/donate/. There are no "shared" crowdfunding effort yet because it can get complicated to handle internationally and equitably.

That said, I don't see a "switch in direction to focus on OpenXR integration" happening anytime soon, because the direction currently is towards some common grounds that would be useful for 2d, 3d _and_ xr. Also, the sponsorship pledge linked above specifically protects the project direction from money investment.

On a more positive note, progress towards XR is happening, there are discussions on the [#xr-dev discord channel](https://discord.com/channels/691052431525675048/829769375618826251), and there's definitely community interest. You could also get in touch with someone accepting sponsorship and see if that's something they would be interested to work on, though without a guarantee those contributions would be accepted if they don't gather community support.
